taurus/entity/bounds/tests/test_real_bounds.py

```diff
@@ -73,9 +43,4 @@ def test_constructor_error():
 def test_type_mismatch():
     """Test that incompatible types cannot be matched against RealBounds."""
     bounds = RealBounds(0, 1, default_units="meters")
-    assert not bounds.validate(NominalCategorical("foo"))
     assert not bounds.contains(IntegerBounds(0, 1))
-    with pytest.raises(TypeError):
-        bounds.validate(0)
-    with pytest.raises(TypeError):
-        bounds.contains([.33, .66])
```

Comment:
I disagree with removing this

Reply:
This leaves `contains`, so I should have left this check for a type error on `contains`; nice catch!